.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+
+jobs:
+
+  release:
+    # Should match the version that Polygott uses.
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: sudo apt-get install -y libpulse-dev libopus-dev libmp3lame-dev xz-utils
+
+      - name: Bump version and push tag
```

Comment:
woah, this is neat

Reply:
it's so amazing! i no longer have to think about releases in most of my OSS repositories n_n